docs/articles/documentation/using-testcafe/common-concepts/screenshots-and-videos.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: docs
+title: Screenshots and Videos
+permalink: /documentation/using-testcafe/common-concepts/screenshots-and-videos.html
+---
+# Screenshots and Videos
+
+TestCafe allows you to take screenshots of the webpage under test and record videos of test runs.
+
+* [Screenshots](#screenshots)
+  * [Prerequisites for Screenshots](#prerequisites-for-screenshots)
+  * [Enable Screenshots](#enable-screenshots)
+  * [Take Screenshots at Arbitrary Moments During Test Run](#take-screenshots-at-arbitrary-moments-during-test-run)
+  * [Take Screenshots When a Test Fails](#take-screenshots-when-a-test-fails)
+* [Record Videos](#record-videos)
+  * [Prerequisites for Video Recording](#prerequisites-for-video-recording)
+  * [Enable Video Recording](#enable-video-recording)
+  * [Basic Video Options](#basic-video-options)
+  * [Video Encoding Options](#video-encoding-options)
+* [Where Screenshots and Videos Are Saved](#where-screenshots-and-videos-are-saved)
+  * [Default Path Patterns](#default-path-patterns)
+  * [Custom Path Patterns](#custom-path-patterns)
+    * [Screenshot Custom Path Pattern](#screenshot-custom-path-pattern)
+    * [Video Custom Path Pattern](#video-custom-path-pattern)
+  * [Path Pattern Placeholders](#path-pattern-placeholders)
+
+## Screenshots
+
+TestCafe allows you to take screenshots of the webpage under test at any moment during test run, or automatically whenever a test fails.
```

Comment:
TestCafe allows you to take screenshots of the tested webpage and record videos of test runs.

Reply:
It seems like you've pasted a sentence from the previous reply here (I judge by the 'videos' being mentioned). What's the right version? Is it:

*TestCafe allows you to take screenshots of the tested webpage at any moment during test run, or automatically whenever a test fails.*

Reply:
I agree with your version